src/XamlX/TypeSystem/TypeSystem.cs

```diff
@@ -45,7 +45,7 @@ interface IXamlType : IEquatable<IXamlType>
 #if !XAMLX_INTERNAL
     public
 #endif
-    interface IXamlParameterInfo : IXamlMember
+    interface IXamlParameterInfo
```

Comment:
Is it intended?

Reply:
Yes, as parameter isn't really a type member. For example in SRE `ParameterInfo` doesn't derive from `MemberInfo` at all, and `DeclaringType` can't be implemented.

Sorry, I've forgot to add this to the notes!